docs/datetime.txt

```diff
@@ -201,25 +201,24 @@ However, in some applications it is important that
 return different times, as they would normally if the application is
 run anywhere other than the UTC timezone.
 
-The best way to understand how to use
-:func:`~testfixtures.test_datetime` in these situations is to think of
-the internal queue as being a queue of :class:`~datetime.datetime`
-objects at the current local time with a `tzinfo` of None, much as
-would be returned by :meth:`~datetime.datetime.now`. 
-If you pass in a `tz` parameter to
+For these situations, you can explicitly specify what timezone the
+internal queue of :class:`~datetime.datetime` objects should be
+interpreted in by passing in a `tzinfo` to
+:func:`~testfixtures.test_datetime`. (When not specified, this
+defaults to UTC -- which is why both :meth:`~tdatetime.now` and
+:meth:`~tdatetime.utcnow` return the same values in the simple case
+where a `tzinfo` is not specified.)
+
+In this case, :meth:`~tdatetime.now` will return the internal time as
+a naive time in the local timezone, and :meth:`~tdatetime.utcnow` will
+first convert the internal time to UTC, and then return that as a naive
+time. Just as the corresponding :class:`~datetime.datetime` methods do.
+
+Similarly, if you pass in a `tz` parameter to
 :meth:`~tdatetime.now` it will be applied to the value
 before it is returned in the same way as it would by
 :meth:`datetime.datetime.now`.
 
-If you pass in a `tzinfo` to :func:`~testfixtures.test_datetime`, this
-will be taken to indicate the timezone you intend for the local times
-that :meth:`~tdatetime.now` simulates.
-As such, that timezone will be used to compute values returned from
-:meth:`~tdatetime.utcnow` such that they would be :class:`test_datetime`
-objects in the UTC timezone with the `tzinfo` set to ``None``, as
-would be the case for a normal call to
-:meth:`datetime.datetime.utcnow`.
```

Comment:
This is also correct, but not as clear. I wonder if these two paragraphs need moving higher up rather than removing?